lib/game_logic.rb

```diff
@@ -0,0 +1,103 @@
+require_relative '../lib/board'
+module WinNumbers
+  WIN_COMBINATIONS = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [6, 4, 2],
+    [0, 4, 8]
+  ].freeze
+end
```

Comment:
## Module winNumbers array
It's not a great idea to mixed all inside the same file because when an application increase it's difficult to maintain 
- Move this module o another file
- Or what the most right way to fix this is to add win combinations array in the Game Logic class